pkg/config/config.go

```diff
@@ -42,9 +42,9 @@ type (
 	}
 
 	CloudResourceLink struct {
-		Source string `json:"source,omitempty" yaml:"source,omitempty" toml:"source,omitempty"`
-		Target string `json:"target,omitempty" yaml:"target,omitempty" toml:"target,omitempty"`
-		Type   string `json:"type,omitempty" yaml:"type,omitempty" toml:"type,omitempty"`
+		Source core.ResourceId `json:"source,omitempty" yaml:"source,omitempty" toml:"source,omitempty"`
```

Comment:
Are Links used for anything right now? How do they fit in with the current direction?